### ISSUE TITLE
Deactivate Lease button missing context

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6784]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.1), the `Deactivate Lease` buttons on the `Expired Active Leases` tab are in table cells which are missing context and do not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/leases?locale=en#expired and run the Site Improve browser extension

![missing context lease](https://github.com/samvera/hyrax/assets/32644254/e597b2fc-c791-445a-8c6e-1863cd6471b9)

### Related Issues
#6782

**Comment by christhepianist** [comment]
Tested and confirmed.

**Comment by aahurford** [comment]
fixed in pg and dev
